.github/workflows/build.yml

```diff
@@ -119,7 +128,7 @@ jobs:
       - name: Install Compiler
         run: |
           apt-get update
-          apt-get install -y sudo   # sudo is not present on the image
+          apt-get install -y man sudo   # sudo is not present on the image
```

Comment:
Why sudo is needed?

Reply:
legacy purposes, because the install script used to require it. Removed and build is still passing :)